explore/src/main/scala/explore/tabs/ObsGroupTiles.scala

```diff
@@ -34,85 +33,53 @@ case class ObsGroupTiles(
 ) extends ReactFnProps(ObsGroupTiles.component)
 
 object ObsGroupTiles:
-
   private type Props = ObsGroupTiles
 
   val component = ScalaFnComponent
     .withHooks[Props]
     .render: props =>
-
-      val tiles =
-        if !props.groups.get.contains(props.groupId.asRight) then List.empty
-        else
-          val groupTreeKey = props.groupId.asRight
-          // First zoom to the node, to get the number of child elements
-          // We can safely .get here because we know the key is in the tree and that the value is a Grouping
-          val node         = props.groups
-            .zoom(
-              _.getNodeAndIndexByKey(groupTreeKey).get,
-              modF =>
-                groups =>
-                  groups
-                    .getNodeAndIndexByKey(groupTreeKey)
-                    .map(modF)
-                    .map((newValue, newIndex) =>
-                      groups.upserted(groupTreeKey, newValue.value, newIndex)
-                    )
-                    .getOrElse(groups)
+      val editTile: Tile[?] =
+        Tile(
+          GroupEditTileIds.GroupEditId.id,
+          s"${
+              if props.group.get.system then props.group.get.name.foldMap(_.value)
```

Comment:
Does this no longer append the type of group after the name?

Reply:
Not for system groups. This logic was already in place.